server/routes/service-provider/action-plan/appointment/post-session-feedback/attendance/postSessionAttendanceFeedbackPresenter.test.ts

```diff
@@ -7,7 +7,7 @@ describe(PostSessionAttendanceFeedbackPresenter, () => {
     it('contains a title including the name of the service category and a subtitle, and the attendance questions', () => {
       const appointment = actionPlanAppointmentFactory.build()
       const serviceUser = deliusServiceUserFactory.build({ firstName: 'Alex' })
-      const presenter = new PostSessionAttendanceFeedbackPresenter(appointment, serviceUser)
+      const { presenter } = new PostSessionAttendanceFeedbackPresenter(appointment, serviceUser)
```

Comment:
slightly confused about this interface. if you have to do PostSessionAttendanceFeedbackPresenter.presenter to get the presenter, what is PostSessionAttendanceFeedbackPresenter?

Reply:
I will reword PostSessionAttendanceFeedbackPresenter to: `ActionPlanSessionPostSessionAttendanceFeedbackPresenter`